Added information about Licences, and Copyright in Chapter 23. (#5)
VegaDeftwing/opinionatedguide: Engineering/23-fosscopyright.md

- [Eclipse Public License (EPL)](https://en.wikipedia.org/wiki/Eclipse_Public_License)
- [MIT License](https://en.wikipedia.org/wiki/MIT_License)

## What is Copyright?

Copyright is a type of intellectual property that gives its owner the exclusive right to make copies of a creative work, usually for a limited time. The creative work may be in a literary, artistic, educational, or musical form. Copyright is intended to protect the original expression of an idea in the form of a creative work, but not the idea itself. A copyright is subject to limitations based on public interest considerations. It is widely believed that copyrights are a must to foster cultural diversity and creativity.

## Why Copyright is bad?

> Copyright is acceptable when it is quantifiable but many products are abstract like music, digital media. So keeping the contract fair and simple is pretty hard. Copyright is bad in terms of creativity and freedom. Because of copyright, people are restricted and their hands are tied to avoid any overlap with others. The problem turns out to be the fact that the central features of copyright are polar opposite to the things that support creativity.

Reviewer: This needs changed. Copyright is, at the very least, a trade-off of freedoms. Copyright is necessary to motivate people to make art, as without it they can not earn money from anything they make. Being a musician or photographer requires a ton of expensive equipment, they need that promise back that they can earn money. This paragraph makes it seem wholly bad.

> Creativity is a tricky thing to understand, and more trickier to understand what animates the creative spark and why some people are more creative than others. But when creativity is quantified in money because of the ownership of that creative idea, creativity loses traction.

Reviewer: This needs changed for the same reason as above, claims that copyright is only bad is a problem. There are also multiple grammar issues, I'm fixing those in this suggestion for your reference, but I suspect these two lines will need re-written outright.
```suggestion
Creativity is complicated, more so when you try to understand what motivates the creative spark or why some people are more creative than others. When creativity is rewarded monetarily because of the ownership of that creative idea, creativity loses traction.
```

> The reason for this? Creativity is closely linked to motivation, and humans become creative when they’re internally motivated by curiosity or interest or desire. They get demotivated and less creative, when you introduce money into the equation. Many people believe licenses and copyright is holding the creativity of humans and lacking any growth in that sector.

Reviewer: > They get demotivated and less creative, when you introduce money into the equation.

Nobody is demotivated by money. Even when I often turn down money for doing a job it's because I don't want to deal with the possible tax consequences. 

Grammar: 
```suggestion
The reason for this? Creativity is closely linked to motivation, and humans become creative when they’re internally motivated by curiosity, interest, or desire. They get demotivated and less creative, when you introduce money into the equation. Many people believe licenses and copyright are holding back the creativity of humans.
```

> Many people believe

This phrase should be avoided in general. https://en.wikipedia.org/wiki/Argumentum_ad_populum

Reviewer: also, this makes the page contradict itself as above in 'What is Copyright?' you have
> "It is widely believed that copyrights are a must to foster cultural diversity and creativity."
